src/Processors/Formats/Impl/CSVRowInputFormat.cpp

```diff
@@ -133,6 +133,8 @@ static void skipEndOfLine(ReadBuffer & in)
                 "Cannot parse CSV format: found \\r (CR) not followed by \\n (LF)."
                 " Line must end by \\n (LF) or \\r\\n (CR LF) or \\n\\r.");
     }
+    else if (!in.eof() && ignore_input_fields)
+        skipToNextLineOrEOF(in);
```

Comment:
Maybe this will not work if the field is in quotes and contains a line break

Reply:
OK, I will test this and your pr whether solve my problems. thanks.